docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -262,6 +262,63 @@ The default is `false`.
 Path to the validator client data. The default is `<data-base-path>/validator` where `<data-base-path>`
 is specified using [`--data-base-path`](#data-base-path-data-path).
 
+### ee-endpoint
+
+=== "Syntax"
+
+    ```bash
+    --ee-endpoint=<URL>
+    ```
+
+=== "Example"
+
+    ```bash
+    --ee-endpoint=http://localhost:8545
+    ```
+
+=== "Environment variable"
+
+    ```bash
+    TEKU_EE_ENDPOINT=http://localhost:8545
```

Comment:
I think the JSON-RPC port for the engine APIs differs to the default 8545, perhaps we should therefore use a different port here. Perhaps check what the ports are used in Besu for the `--engine...` CLI options.

Reply:
Or maybe I'm missunderstanding

Reply:
fixed